keyboards/cmm_studio/saka68/readme.md

```diff
@@ -0,0 +1,14 @@
+# CMM.Studio Saka68
+
+CMM.Studio Saka68 Keyboard
+
+![CMM.Studio Saka68](https://i.imgur.com/gZ9Thjel.png)
+
+* Keyboard Maintainer: [CMM.Studio Freather](https://github.com/CMMS-Freather)
+* Hardware Supported: PCB, Atmega32u4
+
+Make example for this keyboard (after setting up your build environment):
+
+    make cmm_studio/saka68:default
+
```

Comment:
Can you add "clear instructions on how to reset the board into bootloader mode" as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

Reply:
> Can you add "clear instructions on how to reset the board into bootloader mode" as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

updated